src/Halogen/Svg/Attributes/CSSLength.purs

```diff
@@ -0,0 +1,40 @@
+module Halogen.Svg.Attributes.CSSLength
+  ( CSSLength(..)
+  ) where
+
+import Prelude
+
+data CSSLength
+  = Cm Number
+  | Mm Number
+  | Inches Number
+  | Px Number
+  | Pt Number
+  | Pc Number
+  | Em Number
+  | Ex Number
+  | Rem Number
+  | Vw Number
+  | Vh Number
+  | Vmin Number
+  | Vmax Number
+  | Pct Number
+  | Nil
+
+instance showCSSLength :: Show CSSLength where
+  show = case _ of
+    Cm i     -> show i <> "cm"
```

Comment:
I feel like there should be a `printCssLength` function somewhere here. Perhaps you've added it in a commit I haven't seen yet?

Reply:
I agree. And the same goes for FontSize. I left it out of the reformatting PR since there was already a lot going on. I'm planning to address this when I resolve #22. I'm planning on doing #22 in two phases:
Phase 1: Introduce printCSSLength and printFontSize. Also implement Show (and derive Eq?) instances for any attribute type that doesn't have them using the standard Show convention (that Show produces valid code). This is not a breaking change.
Phase 2: Change showCSSLength and showFontSize to the standard convention. This is a breaking change, albeit a minor one. Perhaps this gets lumped in with other more significant breaking changes (like CamelCase) so that we have few breaking events.

Reply:
Oh, right. We're reformatting things in this PR, nothing else.